src/coprocessor/codec/convert.rs

```diff
@@ -375,6 +382,65 @@ pub fn convert_json_to_uint(ctx: &mut EvalContext, json: &Json, tp: FieldTypeTp)
     convert_uint_to_uint(ctx, val, tp)
 }
 
+/// Converts a `DateTime` to printable string representation
+#[inline]
+pub fn convert_datetime_to_numeric_string(dt: &DateTime) -> String {
```

Comment:
Maybe better to implement some function like `to_numeric_string(&self)` in `Time`.

Reply:
Could you provide a reference TiDB implementation where this function comes from?